### ISSUE TITLE
SRS specification

### ISSUE THREAD

**Issue by connormanning** [connormanning/entwine#131]
Right now the SRS is exposed as a WKT string.  Instead, after some discussion with @hobu, I propose  a JSON object with "authority", "horizontal", and "vertical" (if applicable) keys.  For example:
```
"srs": { "authority": "EPSG", "horizontal": "3857" }
"srs": { "authority": "EPSG", "horizontal": "3857", "vertical": "4326" }
```

See connormanning/ept#1.

**Comment by msmitherdc** [comment]
This would mean you won’t support projections that are epsg codes.

**Comment by hobu** [comment]
If you don't have a code, you can still add a `wkt` entry. Clients have the option to consume the WKT, but consumption of WKT is not going to be required for clients to be able to consume EPT. If they can speak authority/code, they're good.

**Comment by connormanning** [comment]
Closed via #139.
